FIAT/dual_set.py

```diff
@@ -54,13 +57,67 @@ def to_riesz(self, poly_set):
         tshape = self.nodes[0].target_shape
         num_nodes = len(self.nodes)
         es = poly_set.get_expansion_set()
+        ed = poly_set.get_embedded_degree()
         num_exp = es.get_num_members(poly_set.get_embedded_degree())
+        nbf = poly_set.get_num_members()
 
         riesz_shape = tuple([num_nodes] + list(tshape) + [num_exp])
 
         self.mat = numpy.zeros(riesz_shape, "d")
```

Comment:
I realise this is inherited code, but `mat` is a datatype, it's not really a great name for a member.

Reply:
Will change to self.matrix.  It only appears in this file.